fruit_classifier/preprocessing/preprocessing_utils.py

```diff
@@ -6,6 +6,53 @@
 from fruit_classifier.utils.file_utils import copytree
 
 
+def truncate_filenames(raw_dir):
+    """
+
+    Parameters
+    ----------
+    raw_dir: Will truncate files in raw_dir's subfolders
+
+      # Assumes raw_dir is a directory that contains ONLY directories,
```

Comment:
This should be moved under the one-liner (after a blank line) and without the `#`